symbols/arm9.yml

```diff
@@ -12387,9 +12428,9 @@ arm9:
         NA: 0x20
         JP: 0x20
       description: |-
-        Could be related to missions
+        A list of functions used to validate missions of different types at 02060CEC [EU].
         
-        Note: unverified, ported from Irdkwia's notes
+        type: function[8]
```

Comment:
Can we remove this too? IMO both the symbol name and description make it obvious enough what this is, even if the C header doesn't explicitly list a type.